front/src/modules/pathfinding/utils.ts

```diff
@@ -56,7 +56,7 @@ export const getPathfindingQuery = ({
       if ('track' in step) {
         return {
           track: step.track,
-          offset: step.offset,
+          offset: step.offset * 1000,
```

Comment:
If you could just add a comment for maintainability

```suggestion
         // pathfinding blocks endpoint requires offsets in mm
          offset: step.offset * 1000,
```